man/systemd.target.xml

```diff
@@ -99,6 +99,24 @@
   </refsect1>
 
   <refsect1>
+    <title>Example</title>
+
+    <example>
+      <title>Simple target</title>
+
+      <programlisting># emergency-net.target
+[Unit]
+Description=Emergency with networking
+Requires=emergency.target
+After=emergency.target
+AllowIsolate=yes</programlisting>
+
+      <para>Units can now have a <varname>WantedBy=</varname> value
```

Comment:
That part is misleading. WantedBy can only appear in the [Install] section, so it'd be better to reword this as "Other units can have WantedBy=emergency-net.target in the [Install] section. After they are enabled, they will be started before emergency-net.target is started."

Also, shouldn't your emergency-net.target depend on some networking units? Right now the example seems incomplete, because it doesn't yet differ from emergency.target in any way.

Also, since this target seems to be designed to be used as emergency.target replacement, it'd be nice to give an example of how to actually use it.